tests/test_version.py

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright 2024 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from version_tag_check.version import Version
```

Comment:
Please in this whole file check not used modules and pars. Check two extra lines between the methods as well.

Reply:
I did provide you more info. It is up to you to handle the spaces:) closing.

Reply:
I would like to keep one empty line between two test methods in same group.